src/crypto/chacha20.cpp

```diff
@@ -23,6 +24,8 @@ static const unsigned char tau[] = "expand 16-byte k";
 
 void ChaCha20::SetKey(const unsigned char* k, size_t keylen)
```

Comment:
This function's declaration is commented:

```
    void SetKey(const unsigned char* key, size_t keylen); //!< set key with flexible keylength; 256bit recommended */
```

That should be updated to say that `keylen` is in bytes, and that only 16 and 32 are valid.

Reply:
The ctor which takes a keylen should also be commented to document assumptions.

Reply:
Done.